mycore-webtools/src/main/vue/processing/index.html

```diff
@@ -3,6 +3,14 @@
 <head>
     <meta charset="UTF-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <!-- <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css"
```

Comment:
do you need this commented HTML here?

Reply:
Its handy for local developing. I just need to uncomment it.